README.md

```diff
@@ -105,7 +107,6 @@ beyond an experimental nature, please submit a support request and we
 can reconsider adding it back in.
 
    - Hadoop 1.X support
-   - Tachyon
```

Comment:
lets just leave the Tachyon removal documentation, we'll just say we added Alluxio back instead